src/main/java/com/evgeniyfedorchenko/animalshelter/backend/entities/Animal.java

```diff
@@ -1,9 +1,51 @@
 package com.evgeniyfedorchenko.animalshelter.backend.entities;
 
 
-//@Entity
+import jakarta.annotation.Nullable;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Entity
+@Table(name = "animals")
 public class Animal {
 
-//    Сущность животного, которое содержится в питомнике, либо у усыновителя
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @EqualsAndHashCode.Include
+    private long id;
+
+    @NotBlank(message = "Animal's name should not be blank")
+    @Size(max = 30)
+    private String name;
+
+    @NotNull(message = "Animal's 'isAdult' should not be null")
+    private boolean isAdult;
```

Comment:
```suggestion
    private boolean adult;
```
поля без is, это методы, которые возвращают boolean, надо называть на is